docs/asciidoc/static/breaking-changes.asciidoc

```diff
@@ -21,11 +21,11 @@ Be sure to specify the correct value for the `--version` option during installat
 [float]
 ==== Configuration Changes
 
-The Elasticsearch output plugin configuration has the following changes:
+The [Elasticsearch output plugin](https://www.elastic.co/guide/en/logstash/2.0/plugins-outputs-elasticsearch.html) configuration has the following changes:
 
 * The `host` configuration option is now `hosts`, allowing you to specify multiple hosts and associated ports in the 
 `myhost:9200` format
-* New options: `bind_host`, `bind_port`, `cluster`, `embedded`, `embedded_http_port`, `port`, `sniffing_delay`
+* New options: `sniffing_delay`
```

Comment:
The proper asciidoc syntax is Elasticsearch 
https://www.elastic.co/guide/en/logstash/2.0/plugins-outputs-elasticsearch.html[output plugin], and I'd rather link to *current* and not hardlink to 2.0, so please replace with 

> `The Elasticsearch https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html[output plugin] configuration has the following changes:`